packages/alfa-html/src/alphabet.ts

```diff
@@ -1573,3 +2160,15 @@ function startsWith(
 
   return true;
 }
+
+function findAppropriateState(state: State) {
```

Comment:
I'd better qualify the name of this function as it's very specific to script patterns.